src/System/Hatrace.hs

```diff
@@ -2389,7 +2430,10 @@ data FileWriteEvent
 -- NOTES:
 -- * only calls to `write` are currently used as a marker for writes and syscalls
 --   `pwrite`, `writev`, `pwritev` are not taken into account
-fileWritesConduit :: (MonadIO m) => ConduitT (CPid, TraceEvent) (FilePath, FileWriteEvent) m ()
+fileWritesConduit ::
+     (MonadIO m)
+  => ConduitT (CPid, TraceEvent (Syscall, SyscallArgs)) ( FilePath
+                                                        , FileWriteEvent) m ()
```

Comment:
I'd probably reflow it to

```
ConduitT
  (CPid, TraceEvent (Syscall, SyscallArgs))
  ( FilePath, FileWriteEvent)
  m
  ()
```

so that it's not length-dependent.